include/evmc/evmc.h

```diff
@@ -80,8 +80,7 @@ enum evmc_call_kind
     EVMC_CALLCODE = 2,     /**< Request CALLCODE. */
     EVMC_CREATE = 3,       /**< Request CREATE. */
     EVMC_CREATE2 = 4,      /**< Request CREATE2. Valid since Constantinople.*/
-    EVMC_CREATE3 = 5,      /**< Request CREATE3. Valid since Prague.*/
-    EVMC_CREATE4 = 6       /**< Request CREATE4. Valid since Prague.*/
+    EVMC_CREATE34 = 5      /**< Request CREATE3 or CREATE4. Valid since Prague.*/
```

Comment:
Maybe `EVMC_CREATE3_4`?

Reply:
indeed, it is a bit better. If no one objects, I'll change to that